app/models/user/User.scala

```diff
@@ -366,6 +384,20 @@ class UserDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
       parsed <- parse(result)
     } yield parsed
 
+  def findPayingOrgaIdForMultiUser(multiUserId: ObjectId): Fox[Option[ObjectId]] =
+    for {
+      rows <- run(q"""SELECT _organization
+                      FROM $existingCollectionName
+                      WHERE _multiUser = $multiUserId
+                      AND NOT isDeactivated
+                      AND _organization IN (
+                        SELECT _id FROM webknossos.organizations
+                        WHERE pricingPlan IN ('Team', 'Power', 'Custom')
```

Comment:
maybe pass the wk enums in here rather than strings, so that this will pop up as usages when looking with the IDE